strongbox-commons/src/main/java/org/carlspring/strongbox/artifact/coordinates/ArtifactLayoutLocator.java

```diff
@@ -18,9 +16,9 @@
 
     private static final Logger logger = LoggerFactory.getLogger(ArtifactLayoutLocator.class);
 
-    private volatile static Map<String, Class<? extends ArtifactCoordinates>> layoutEntityMap;
+    private volatile static Map<String, ArtifactLayoutDescription> layoutEntityMap;
 
-    public static Map<String, Class<? extends ArtifactCoordinates>> getLayoutEntityMap()
+    public static Map<String, ArtifactLayoutDescription> getLayoutEntityMap()
```

Comment:
I'm sorry, but... what is an `ArtifactLayoutLocator`? Would you mind adding some javadocs/comments for this?

Reply:
done